common/lib/xmodule/xmodule/video_module/video_module.py

```diff
@@ -861,6 +862,15 @@ def student_view_data(self, context=None):
                     "file_size": 0,  # File size is unknown for fallback URLs
                 }
 
+        # Include youtube link if there is no encoding for mobile- ie only a fallback URL or no encodings at all
+        # We are including a fallback URL for older versions of the mobile app that don't handle Youtube urls
+        if self.youtube_id_1_0 and "mobile_low" not in encoded_videos:
+            youtube_1_0_url = "https://www.youtube.com/watch?v={}".format(self.youtube_id_1_0)
```

Comment:
Look at the existing code in this module for creating youtube urls.  In `get_context` and `get_youtube_link`.  You can consider refactoring that code for reusability.  That other code uses `'http://youtu.be/{0}'.format(video_id)` for creating the youtube url.